homework-3/script-1.js

```diff
@@ -0,0 +1,24 @@
+const YourName = prompt("What's Your Name?");
+const YourSurname = prompt("What's Your Surname?");
+const YourFavoriteNumber = prompt("What's Your Favorite Number");
+console.log(YourName);
+console.log(YourSurname);
+console.log(YourFavoriteNumber);
+console.log(`"Hello,${YourName} ${YourSurname}! Your Favorite Number Is ${YourFavoriteNumber}"`);
+const FirstNumber = +prompt("Enter Please The First Number");
+const SecondaryNumber = +prompt("Enter Please The Secondary Number");
+const Sum = FirstNumber + SecondaryNumber;
+console.log(Sum);
+const Difference = FirstNumber - SecondaryNumber;
+console.log(Difference);
+const Multiplication = FirstNumber * SecondaryNumber;
+console.log(Multiplication);
+const Division = FirstNumber / SecondaryNumber;
+console.log(Division);
+const UserAge = prompt("What year were you born?");
+console.log("User year's born is", UserAge);
+const Remainder = FirstNumber % SecondaryNumber;
+console.log(`"Remainder of a number"${FirstNumber} by number ${SecondaryNumber} equal ${Remainder}`);
+const ThirdNumber = +prompt("Enter Please The Third Number For Pairing Checks");
+const PairingNumber = ThirdNumber % 2 === 0;
```

Comment:
For obviously boolean values common practice is to use variable names that start with is or has, in your case, `isPairingNumber` sounds better.